### ISSUE TITLE
Array return misidentifies static attributes

### ISSUE THREAD

**Issue by mgarber93** [jridgewell/babel-plugin-transform-incremental-dom#96]
When [returning an array](https://github.com/mgarber93/babel-plugin-transform-incremental-dom/blob/array-return-no-arguments/test/fixtures/issues/95/actual.js), the transpiled code returns an array with only the first element invoked with arguments. [The rest of the array attempts to access the arguments via a statics array (see line 16 and 33: Uncaught ReferenceError: bar is not defined).](https://github.com/mgarber93/babel-plugin-transform-incremental-dom/blob/array-return-no-arguments/test/fixtures/issues/95/expected.js)

Strangely, I have been able to work around this issue by trying to use [tagged templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). This jsx:
```javascript
    ...
    const workAround = t => t;
    return ([
      <span
        foo={ workAround`My bar is: ${bar}`}
      >one</span>,
      <span
        foo={ workAround`My bar is: ${bar}`}
      >two</span>,
      <span
        foo={ workAround`My bar is: ${bar}`}
      >three</span>
    ]);
    ...
```
transpiles to return an array where each jsxWrapper is invoked with the correct arguments:
```javascript
return [_jsxWrapper(_span$wrapper, [workAround(_templateObject, bar)]), _jsxWrapper(_span$wrapper2, [workAround(_templateObject, bar)]), _jsxWrapper(_span$wrapper3, [workAround(_templateObject, bar)])];
```

**Comment by jridgewell** [comment]
Should be fixed in v4.2.1

**Comment by mgarber93** [comment]
Looks good, thanks @jridgewell!
